tools/packaging/static-build/qemu/build-qemu.sh

```diff
@@ -16,7 +16,7 @@ kata_static_build_scripts="${kata_static_build_dir}/scripts"
 
 git clone --depth=1 "${QEMU_REPO}" qemu
 pushd qemu
-git fetch --depth=1 origin "${QEMU_VERSION}"
+git fetch --depth=1 origin "${QEMU_BRANCH}:${QEMU_VERSION}"
```

Comment:
This does not work as you intended: this fetches the ref called `${QEMU_BRANCH}` from origin and names it `${QEMU_VERSION}` locally.
So for the vanilla `qemu` build it tries to build tip of the master branch, that's why CI does not pass.